app/views/allocations/_deallocate_pom.html.erb

```diff
@@ -5,7 +5,11 @@
     <% end %>
   </h2>
   <p>
-  &nbsp;
+    <% unless allocation.event_trigger == 'offender_transferred' %>
```

Comment:
This raises the interesting question of what we should display when someone is transferred - the history will look strange maybe?